cowtown2023/src/main/java/com/team2357/frc2023/subsystems/ShooterSubsystem.java

```diff
@@ -3,17 +3,42 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import com.team2357.frc2023.Constants;
+import com.team2357.frc2023.Constants.SHOOTER;
+import com.team2357.lib.subsystems.LimelightSubsystem;
+import com.team2357.lib.util.RobotMath;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ShooterSubsystem extends SubsystemBase {
 
     private CANSparkMax m_topShooterMotor;
     private CANSparkMax m_bottomShooterMotor;
+    private SHOOTER m_config;
```

Comment:
We're not using configs this year so remove everything relating to them. Anything that you would've been in a config can be put straight into constants.